30-indoor-air-quality-and-gas-leak/README.md

```diff
@@ -17,65 +18,26 @@ Monitor air quality, including Air Quality Index, CO2 and VOC concentration, and
 
 Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/#set-up-notehub).
 
-## Sparrow Setup
-
-### Quickstart
-
-Follow the [Sparrow Quickstart](https://dev.blues.io/quickstart/sparrow-quickstart/) pair your Sparrow reference node with the gateway and associate it with the Notehub project you just created. Note also that you only need one reference node for this project, so you don't need to pair both nodes that came with the dev kit. Make sure that you use the ProductUID generated in [Notehub Setup](#notehub-setup) when it comes time to issue the [`hub.set`](https://dev.blues.io/api-reference/notecard-api/hub-requests/#hub-set) command in the quickstart.
-
-After you've completed the quickstart, leave the Notecarrier and Sparrow Essentials board powered and connected. These two devices will act as our gateway to Notehub, and you won't need to touch them again. The rest of this guide will focus on the Sparrow Reference Sensor node.
+## Swan Setup
```

Comment:
You need to have steps in here about setting up the gateway, and then connecting the Notecard + Swan + Notecarrier F.

You can link to the docs for each of those steps (https://dev.blues.io/lora/connecting-to-a-lorawan-gateway/ and https://dev.blues.io/guides-and-tutorials/collecting-sensor-data/notecarrier-f/blues-swan/c-cpp-arduino-wiring/#set-up-hardware probably), but they should be in here.